corehq/apps/reports/static/reports/js/maps_utils.js

```diff
@@ -82,18 +81,18 @@ hqDefine("reports/js/maps_utils", function () {
     function MetricModel(data, parent, root) {
         var m = this;
 
-        this.metric = null;
-        this.name = ko.observable();
-        this.children = ko.observableArray();
-        this.expanded = ko.observable(false);
-        this.selected = ko.observable(false);
-        this.parent = parent;
+        m.metric = null;
```

Comment:
was `m` used at all before?

Reply:
It's used [down here](https://github.com/dimagi/commcare-hq/blob/8b5b05bbae3ccead34bfa6bc9da3981d57b39f55/corehq/apps/reports/static/reports/js/maps_utils.js#L124) within a callback where the value of `this` has changed, I imagine that's why it was created.